x-pack/test/security_solution_cypress/cypress/cypress.config.ts

```diff
@@ -27,6 +27,7 @@ export default defineCypressConfig({
   e2e: {
     experimentalRunAllSpecs: true,
     experimentalMemoryManagement: true,
+    experimentalCspAllowList: ['default-src', 'script-src', 'script-src-elem'],
```

Comment:
I'm a bit of out context here and just curious why we permit CSP directives now? It looks like an orthogonal change in this PR.

Reply:
Hey @maximpn!! In order to be able to execute Cypress we were setting the `csp` parameters when creating the instances for testing in `x-pack/test/security_solution_cypress/config.ts`. In a real (deployed) serverless environment, we don't have a way of doing it, so the above line helps us to execute the tests without facing any issues.